<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:1 rgba(203, 248, 236, 252));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>331</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="horizontalSpacing">
      <number>10</number>
     </property>
     <property name="verticalSpacing">
      <number>10</number>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lblCodigo">
       <property name="font">
        <font>
         <family>Mongolian Baiti</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:black;
</string>
       </property>
       <property name="text">
        <string>Nombre:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txtNombre">
       <property name="font">
        <font>
         <family>Mongolian Baiti</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:5px;
border:1px solid black;
color:black;

font: 12pt &quot;Mongolian Baiti&quot;;</string>
       </property>
       <property name="placeholderText">
        <string comment="Codigo" extracomment="adsdasd">Nombre</string>
       </property>
       <property name="cursorMoveStyle">
        <enum>Qt::LogicalMoveStyle</enum>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblTitulo">
       <property name="font">
        <font>
         <family>Mongolian Baiti</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:black;</string>
       </property>
       <property name="text">
        <string>Platillo:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="txtPlatillo">
       <property name="font">
        <font>
         <family>Mongolian Baiti</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:5px;
border:1px solid black;
color:black;
font: 12pt &quot;Mongolian Baiti&quot;;</string>
       </property>
       <property name="placeholderText">
        <string>Escriba su Platillo</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblCategoria">
       <property name="font">
        <font>
         <family>Mongolian Baiti</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:black;</string>
       </property>
       <property name="text">
        <string>Sede:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboSede">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>Mongolian Baiti</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:black;
font: 12pt &quot;Mongolian Baiti&quot;;</string>
       </property>
       <item>
        <property name="text">
         <string>Seleccione...</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ate</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SantaAnita</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Lima</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Chosica</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>LaVictoria</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Narnia</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lblAutor">
       <property name="font">
        <font>
         <family>Mongolian Baiti</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:black;</string>
       </property>
       <property name="text">
        <string>Direcci√≥n:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="txtDireccion">
       <property name="font">
        <font>
         <family>Mongolian Baiti</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:5px;
border:1px solid black;
color:black;
font: 12pt &quot;Mongolian Baiti&quot;;</string>
       </property>
       <property name="placeholderText">
        <string comment="Autor" extracomment="Autor">Punto de Referencia</string>
       </property>
       <property name="cursorMoveStyle">
        <enum>Qt::LogicalMoveStyle</enum>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btnRegistrar">
         <property name="font">
          <font>
           <family>Mongolian Baiti</family>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:1 rgba(161, 161, 236, 255));
border-radius:7px;
color:white;
border:1px solid black;
}
QPushButton:hover{
background-color:rgb(53,53,79);
border-radius:10px;
}</string>
         </property>
         <property name="text">
          <string>REGISTRAR</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnLimpiar">
         <property name="font">
          <font>
           <family>Mongolian Baiti</family>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:1 rgba(161, 161, 236, 255));
border-radius:7px;
color:white;
border:1px solid black;
}
QPushButton:hover{
background-color:rgb(53,53,79);
border-radius:10px;
}</string>
         </property>
         <property name="text">
          <string>LIMPIAR</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>621</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mongolian Baiti</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;</string>
    </property>
    <property name="text">
     <string>Registre su Pedido, Por favor !</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tblSalida">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>240</y>
      <width>401</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mongolian Baiti</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:1 rgba(161, 161, 236, 255));
color:black;
font: 12pt &quot;Mongolian Baiti&quot;;</string>
    </property>
    <column>
     <property name="text">
      <string>NOMBRE</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>PLATILLO</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>SEDE</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>DIRECCION</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btnBack">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mongolian Baiti</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(170, 170, 255);
border-radius:10px;
border:3px solid black;
color:black;

}
QPushbutton:hover{
background-color:white;
}




</string>
    </property>
    <property name="text">
     <string>BACK</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
